src/riak_cs_acl_utils.erl

```diff
@@ -379,6 +384,38 @@ validate_acl({ok, _}, _) ->
 validate_acl({error, _}=Error, _) ->
     Error.
 
+
+%% @doc Construct an acl. The structure is the same for buckets
+%% and objects.
+-spec acl(string(), string(), string(), [acl_grant()], erlang:timestamp()) -> acl().
+acl(DisplayName, CanonicalId, KeyId, Grants, CreationTime) ->
```

Comment:
That test code tests `acl_from_json`, but the function is not used :no_entry_sign: